package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hash-for-dep",
-  "version": "1.0.2",
+  "version": "1.0.3",
```

Comment:
hmm, ya im unsure if this is major or minor. I guess we never really thought about that. I'll think on it...